<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsersWindow</class>
 <widget class="QWidget" name="UsersWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UsersWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QListView" name="listViewQuestions"/>
     </item>
     <item>
      <widget class="QListView" name="listViewAnswers"/>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="labelAskQuestion">
         <property name="text">
          <string>Add question:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEditAskQuestion"/>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonAskQuestion">
         <property name="text">
          <string>Ask</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditAnswer"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelAnswer">
         <property name="text">
          <string>Add answer: </string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButtonAnswer">
         <property name="text">
          <string>Answer</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
